activesupport/test/cache/stores/null_store_test.rb

```diff
@@ -44,6 +48,7 @@ def test_decrement
   def test_delete_matched
     @cache.write("name", "value")
     @cache.delete_matched(/name/)
+    assert_nil @cache.read("name")
```

Comment:
Presumably this test was implicitly relying on an exception but we really could have this test pass with something like `@cache.delete_matched(/WHATEVER/)` so I decided to also check for the fact that we clean the cache